Car.java

```diff
@@ -0,0 +1,12 @@
+public class Car {
+    int maxSpeed = 130;

```

Comment:
Нарушение Code Convention:
```suggestion
    int maxSpeed = 130;
     
```